src/rules/at-rule-name-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,460 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "@charset\n\"UTF-8\";",
+  }, {
+    code: "@charset\r\n\"UTF-8\";",
+  }, {
+    code: "@import\n\"x.css\";",
+  }, {
+    code: "@import\r\n\"x.css\";",
+  }, {
+    code: "@import\n\"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import\r\n\"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import\nurl(\"x.css\");",
+  }, {
+    code: "@importn\r\nurl(\"x.css\");",
+  }, {
+    code: "@import\nurl(\"x.css\") screen and (orientation:landscape);",
+  }, {
+    code: "@import\r\nurl(\"x.css\") screen and (orientation:landscape);",
+  }, {
+    code: "@namespace\nurl(XML-namespace-URL);",
+  }, {
+    code: "@namespace\r\nurl(XML-namespace-URL);",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px)\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px)\r\nand (orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px) and\n(orientation: landscape) { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and\r\n(orientation: landscape) { }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape)  { }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape)  { }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape)\n{ }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape)\n{ }",
+  }, {
+    code: "@media\n(min-width: 700px) and (orientation: landscape)\r\n{ }",
+  }, {
+    code: "@media\r\n(min-width: 700px) and (orientation: landscape)\r\n{ }",
+  }, {
+    code: "@media\n(min-width: 700px)  and (orientation: landscape) { }",
```

Comment:
@davidtheclark i can remove, but may be false positive if somebody rewrite rule on other logic, be good tested this.